<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoRuns</class>
 <widget class="QMainWindow" name="AutoRuns">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>777</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1221</width>
    <height>777</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoRuns</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_logon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1221</width>
      <height>581</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1221</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>230</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AutoRun Entry</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Publisher</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Image Path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TimeStamp</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Logon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Logon</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Services">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Services</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_services">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1221</width>
      <height>581</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1221</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>230</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AutoRun Entry</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Publisher</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Image Path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TimeStamp</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_drivers">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1221</width>
      <height>581</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1221</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>230</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AutoRun Entry</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Publisher</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Image Path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TimeStamp</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Drivers">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Drivers</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_knowndlls">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1221</width>
      <height>581</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1221</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>230</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AutoRun Entry</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Publisher</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Image Path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TimeStamp</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Knowndlls">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Knowndlls</string>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>650</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>650</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="size">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>620</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="path">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>710</y>
      <width>931</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>931</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="publisher">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>650</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>29</y>
      <width>1221</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QTableWidget" name="tableWidget_tasks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1221</width>
      <height>581</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1221</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>230</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AutoRun Entry</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Publisher</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Image Path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TimeStamp</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Tasks">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Scheduled Tasks</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
